### PR TITLE
style: Inline some things that should never ever appear in a profile.

### PR DESCRIPTION
**Pull request by emilio**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19451)

### CODE REVIEW

File: components/style/gecko/wrapper.rs [hunk 1]
```diff
@@ -236,6 +237,7 @@ impl<'ln> GeckoNode<'ln> {
         true
     }
 
+    #[inline]
     fn flattened_tree_parent(&self) -> Option<Self> {
```
Comment (upsuper) [inline components/style/gecko/wrapper.rs]: I'm not particularly convinced this method should be marked `inline`. It looks non-trivial, especially when `flattened_tree_parent_is_parent` is inlined. And it is called in more than one places. I would say let's just let compiler decide this one.

**Review by upsuper — APPROVED** [review]

File: components/style/gecko/wrapper.rs (hunk 1 above)
Comment (emilio) [reply components/style/gecko/wrapper.rs]: The whole point of `flattened_tree_parent_is_parent` is avoiding the normal ool function call, and this method is _hot_ :(

**Comment by jdm** [comment]
@bors-servo: retry
* sccache error

**Pull request merged**